manifold/src/face_op.cpp

```diff
@@ -115,9 +127,27 @@ void Manifold::Impl::Face2Tri(const VecDH<int>& faceEdge,
       triNormal.push_back(normal);
       triVerts.push_back(tri1);
       triNormal.push_back(normal);
+      meshRelation_.triBary.H().push_back(faceRef.H()[face]);
+      for (int k : {0, 1, 2}) {
+        int index = 0;
+        while (mapping[index] != tri0[k])
+          index += 1;
+        meshRelation_.triBary.H().back().vertBary[k] = halfedgeBary.H()[firstEdge + index];
+      }
+      meshRelation_.triBary.H().push_back(faceRef.H()[face]);
+      for (int k : {0, 1, 2}) {
+        int index = 0;
+        while (mapping[index] != tri1[k])
+          index += 1;
+        meshRelation_.triBary.H().back().vertBary[k] = halfedgeBary.H()[firstEdge + index];
```

Comment:
Perhaps we could make a function out of this block to avoid some code repetition?